test/core/services/core-process-response.service.test.ts

```diff
@@ -180,6 +181,24 @@ describe('CoreProcessResponseService', () => {
           });
         });
       });
+
+      describe('with invalid request token', () => {
+        const response = new Response(JSON.stringify({
+          type: "invalid_request_token",
```

Comment:
I don't think this is enough. If you send `"request_token": null`, you get this error:

```
{
    "type": "invalid_parameters",
    "message": "One or more parameters are invalid: \nproperty '/request_token' is not of type: string"
}
```

Reply:
good call @madejejej 

We're looking to instead change the error type returned from the API in case of these errors, as it doesn't really make sense to search the human-readable message string for token related errors to separate them from other validation errors programmatically.